tests/robot-tests/tests/general_public/methodology_page_absence.robot

```diff
@@ -69,7 +69,7 @@ Search for "specific enquiry"
 
     user clears element text    id:pageSearchForm-input
     user presses keys  specific enquiry   id:pageSearchForm-input
-    user waits until element contains     xpath://*[@id="pageSearchForm-resultsLabel"]   Found 1 result
+    user waits until element contains     id:pageSearchForm-resultsLabel   Found 1 result
```

Comment:
Only needs to be 2 spaces here AFAIK

Reply:
Discussed this with Luke - throughout the UI tests we use more than two spaces to separate arguments and keywords. This can sometimes make it clearer that they're separate. I guess ideally we could keep this the same throughout the test, but it isn't that important so will just leave this.